components/Carousel/Carousel.js

```diff
@@ -1,11 +1,42 @@
 class Carousel {
+    constructor(carouselElement){
+        this.carouselElement = carouselElement
+        this.navLeft = this.carouselElement.querySelector('.left-button');
+        this.navRight = this.carouselElement.querySelector('.right-button');
+        this.images = this.carouselElement.querySelectorAll('img');
+
+        this.navLeft.addEventListener('click', (e) => this.navigate(-1));
+        this.navRight.addEventListener('click', (e) => this.navigate(1));
+        this.navigate(0);
+      
+    }
+    navigate(direction){
+        let counter = 0;
+        let carouselLength = this.carouselElement.length;
+        let currentImg = this.images[0];
+
+        currentImg.classList.remove('current');
+               
+        counter += direction;
+        
+        if(direction === -1 && counter < 0){
```

Comment:
Why check if the direction is -1 and check to see if the counter is negative? Counter is initialized to 0, so if the direction is negative, so it the counter due to the line `counter += direction`.